sources/packages/backend/workflow/src/workflow-definitions/load-assessment-consolidated-data.bpmn

```diff
@@ -136,8 +136,8 @@
           <zeebe:header key="partner1HasTotalIncomeAssistance" value="supportingUsers.Partner1.supportingData.hasTotalIncomeAssistance" />
           <zeebe:header key="partner1BCEAIncomeAssistanceAmount" value="supportingUsers.Partner1.supportingData.bceaIncomeAssistanceAmount" />
           <zeebe:header key="partner1PartnerCaringForDependant" value="supportingUsers.Partner1.supportingData.partnerCaringForDependant" />
-          <zeebe:header key="studentDataExceptionalExpenseAmount" value="data.exceptionalExpenseAmount" />
-          <zeebe:header key="studentDataParents" value="data.parents[].{&#10;  &#34;parentIsAbleToReport&#34;: parentIsAbleToReport,&#10;  &#34;currentYearParentIncome&#34;: currentYearParentIncome&#10;}" />
+          <zeebe:header key="studentDataExceptionalExpenseAmount" value="data.exceptionalExpenseAmount ?? data.exceptionalExpensesApplicationException.exceptionalExpenseAmount" />
+          <zeebe:header key="studentDataParents" value="data.parents[].{&#10;  &#34;parentIsAbleToReport&#34;: parentIsAbleToReport,&#10;  &#34;currentYearParentIncome&#34;: currentYearParentIncome ?? currentYearParentIncomeApplicationException.currentYearParentIncome &#10;}" />
```

Comment:
Found more variables missing backfall.

<img width="899" height="42" alt="image" src="https://github.com/user-attachments/assets/a4ddc59e-fc5c-4bbc-bda7-0677ffe70ada" />

<img width="979" height="46" alt="image" src="https://github.com/user-attachments/assets/b47f326d-aff3-4d8e-847e-7eee32488ad1" />

Reply:
I focused on the inputs and missed that ones for transportation. I will add them.

Reply:
Added